content/recipes/prisma.md

```diff
@@ -1,154 +1,560 @@
+);
+CREATE TABLE "Post" (
+  "id"        INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+  "title"     TEXT    NOT NULL,
+  "content"   TEXT,
+  "published" BOOLEAN DEFAULT false,
+  "authorId"  INTEGER REFERENCES "User"(id)
+);
 ```
 
-#### Integration
+Note that Prisma also features a _schema migration_ tool called [Prisma Migrate](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-migrate). Prisma Migrate lets you manually define your models in your Prisma schema and takes care of creating the tables in your database. Because Prisma Migrate is currently considered experimental, this guide uses an alternative workflow of using plain SQL to create your database tables and generate Prisma models via [introspection](https://www.prisma.io/docs/reference/tools-and-interfaces/introspection). 
 
-Almost done. Now, let's create a module for our Prisma integration.
+##### 5. Introspect your database to obtain your Prisma models in the Prisma schema
+
+Now that you created your database tables, you can _introspect_ the database to generate your [Prisma models](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/models). After that, you will install and generate Prisma Client which will expose queries that are _tailored_ to these models.
+
+To introspect your database, run the following command in your terminal:
+
+```bash
+npx prisma introspect
+```
+
+This reads your SQL schema and translates each table into a corresponding Prisma model. Your `schema.prisma` file now looks as follows:
+
+```prisma
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  email String  @unique
+  id    Int     @default(autoincrement()) @id
+  name  String?
+  Post  Post[]
+}
+
+model Post {
+  authorId  Int?
+  content   String?
+  id        Int      @default(autoincrement()) @id
+  published Boolean? @default(false)
+  title     String
+  User      User?    @relation(fields: [authorId], references: [id])
+}
+```
+
+With your Prisma models in place, you can install and generate Prisma Client.
+
+##### 6. Install and generate Prisma Client
+
+To install Prisma Client in your project, run the following command in your terminal:
+
+```bash
+npm install @prisma/client
+```
+
+Note that this command automatically invokes the `prisma generate` command for you. In the future, you need to run this command after _every_ change to your Prisma models to update your generated Prisma Client.
+
+> **Note** The `prisma generate` command reads your Prisma schema and updates the generated Prisma Client library inside `node_modules/@prisma/client`.
+
+
+##### 7. Use Prisma Client in your NestJS services
+
+You're now able to send database queries with Prisma Client! If you want to learn about the possible queries with Prisma Client, check out the [API documentation](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/crud).
```

Comment:
```suggestion
You're now able to make database requests from Nest with Prisma Client. If you want to learn more about building queries with Prisma Client, check out the [API documentation](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/crud).
```

Reply:
I'm not 100% sure about this one since "database queries" is our official terminology when using Prisma Client so I'd prefer using that instead of "database requests". But I adopted all of your other suggestions in my PR manually 👍